### DISCUSSION TITLE
Port lists on vcpkg.io but install and search do not find

### DISCUSSION THREAD

**Discussion by carlrose** [microsoft/vcpkg#31172] [Q&A]
I attempted to install libavif but got error message Error: while loading port `libavif`: Port definition not found. search also did not find it. However Browse Packages on vcpkg.io shows it and it exists in the ports directory on github.

**Comment by Osyotr** [comment]
Does it exist in your local repo? Is your baseline newer than first release of libavif in vcpkg?

**Reply by carlrose** [reply]
Osyotr
Thanks. I did not realize that I had to pull from vcpkg depository after initial install.
